spec/unit/puppet/provider/ios_interface/test_data.yaml

```diff
@@ -0,0 +1,109 @@
+---
+default:
+  read_tests:
+    "show interface":
+      cli: "!\ninterface GigabitEthernet0/1\n no logging event link-status\n logging event spanning-tree\n logging event subif-link-status\n snmp trap mac-notification change added\n flowcontrol receive desired\n ip dhcp snooping limit rate 1500\n ip dhcp snooping trust\n ip dhcp snooping information option allow-untrusted"
```

Comment:
It would be nice to see the `no logging event trunk-status` in the cli to cover the issue we identified the other day :-) this will make sure it doesn't regress

Reply:
added